server/tserver/src/main/java/org/apache/accumulo/tserver/compactions/Compactable.java

```diff
@@ -76,7 +76,8 @@ public Files(SortedMap<StoredTabletFile,DataFileValue> allFiles,
       var compactingFiles =
           compacting.stream().flatMap(job -> job.getFiles().stream()).collect(Collectors.toSet());
       Preconditions.checkArgument(this.allFiles.containsAll(compactingFiles),
-          "Compacting not in set of all files %s %s", this.allFiles, compactingFiles);
+          "Compacting not in set of all files: %s, compacting files: %s", this.allFiles,
+          compactingFiles);
```

Comment:
The message itself is still confusing to me. How about this?
```suggestion
          "Compacting files %s not in set of all files: %s", compactingFiles,
          this.allFiles);
```

Reply:
That does sound better